styles/App.scss

```diff
@@ -26,6 +26,6 @@
 /* pages */
 @import 'pages/index';
 @import 'pages/committees';
-@import 'pages/events';
+// @import 'pages/events';
```

Comment:
Let's comment this out before merging!

Reply:
oh shoot I think that happened because of me pulling main back into the branch

Reply:
oh no it's still commented - I assume you meant remove, so I'll get rid of it

Reply:
Yup sorry, mistyped!